src/Goerli-DssSpell.t.sol

```diff
@@ -149,12 +149,28 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertTrue(spell.done());
 
         checkIlkClipper(
-            "RETH-A",
-            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
-            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
-            addr.addr("MCD_CLIP_CALC_RETH_A"),
-            OsmAbstract(addr.addr("PIP_RETH")),
-            500 * WAD
+            "GUSD-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_GUSD_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_GUSD_A")),
+            addr.addr("MCD_CLIP_CALC_GUSD_A"),
+            OsmAbstract(addr.addr("PIP_GUSD")),
+            150000 * WAD
```

Comment:
Silly question but when we do checkIlkClipper wouldn't it be better to test with the `dust` amount (15k not 150k) as an edge case check?

Reply:
The dust amount fails probably due to rounding error. I put it to 16k just above.

Reply:
If dust fails then isn't this a fundamental issue with `dust`? Or is the issue unique to the checkIlkClipper test?

Reply:
It's the latter I think. The test should be fixed at some point, but it's not super relevant to this.